sqlglot/expressions.py

```diff
@@ -4053,6 +4056,16 @@ def unit(self) -> t.Optional[Var]:
         return self.args.get("unit")
 
 
+class IntervalOp(TimeUnit):
+    arg_types = {"unit": True, "expression": True}
+
+    def interval(self):
+        return Interval(
+            this=self.expression.copy(),
+            unit=self.unit.copy(),
+        )
```

Comment:
Should we make this a property and call it `to_interval` or something?

Reply:
I'll call it to_interval

Reply:
Hm, `to_interval` isn't quite right, because it doesn't operate on the entire expression. Just the interval part.
And making it a property doesn't feel right either, since this creates a new instance.

Reply:
I see what you mean, feel free to ignore this then.